### PR TITLE
Major changes 1

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made:
- Processing the iso3166.csv file
Since it is said that we need to create the script for downloading the datasets from unstats.un.org website, currently I am busy with fixing the whole codebase, for this case I am currently doing it the same way as the original author's did. (Going to update it as soon as I finish with the current codebase). 
- Added `scripts/config.py` file for easy access to all the **links** and **column_headers**
- Added `scripts/iso3166.py` script for easy creation of `iso3166.json` file
- Updated the `scripts/edgar.py` using selenium for parsing the `sec.gov` website 
  -  The reason I am using the **selenium** because `sec.gov's` sitemap disallows parsing through bs4 or requests, in terms of API it only worls for certain pages which are not available for these paths: `/submit-filings/filer-support-resources/edgar-state-country-codes`, in terms of **GITHUB_ACTIONS** there might be problems opening browser in the `no --headless` mode which are needed for parsing because in `--headless` mode it doesn't work, solution is to use `Xvfb` for setting up the pipeline
 - Updated the `scripts/unterm_names.py` script
 - Updated the `scripts/format_json.py` script

### CODE REVIEW

File: tmp/UNSD-ar.csv
Comment (anuveyatsu) [inline tmp/UNSD-ar.csv:1]: @gradedSystem why do we need these tmp files?

File: scripts/iso3166.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import os
+import pandas as pd
```
Comment (anuveyatsu) [inline scripts/iso3166.py:2]: I don't believe we need to use pandas which is quite huge lib. Also, it isn't listed in the requirements here https://github.com/datasets/country-codes/blob/master/scripts/requirements.pip

File: tmp/UNSD-ar.csv
Comment (gradedSystem) [reply tmp/UNSD-ar.csv:1]: The reason I haven't worked on creating new function to parse csv from the UNSD website in the original `README.md` it is said this:

> NOTE/TODO: currently, preparation requires manual process to download and rename 6 CSV files from https://unstats.un.org/unsd/methodology/m49/overview/

So I just downloaded the most recent files from the UNSD website and used them, going to change after we will run our daily cron jobs

File: scripts/config.py
Comment (anuveyatsu) [inline scripts/config.py:1]: @gradedSystem could you check if we can integrate this into makefile here https://github.com/datasets/country-codes/blob/master/Makefile

File: tmp/UNSD-ar.csv
Comment (anuveyatsu) [reply tmp/UNSD-ar.csv:1]: ok got it @gradedSystem

File: scripts/edgar.py [hunk 2]
```diff
@@ -1,47 +1,51 @@
...
-    name = utils.clean(cells[1].text_content())
-    data.append((code, name))
+import csv
 
-with open('data/edgar.csv', 'wb') as csv_file:
-    csv_writer = csv.writer(csv_file)
-    csv_writer.writerow(header)
-    for row in data:
-        csv_writer.writerow(row)
+from selenium import webdriver
```
[the remark this answers is not in the document]
Comment (gradedSystem) [reply scripts/edgar.py:6]: Yes, I have tried scraping it using all the possible methods and got blocked by their website, also they don't have api for this specific `https://www.sec.gov/submit-filings/filer-support-resources/edgar-state-country-codes` and they disallowed `requests` for certain amount of pages

[the remark this answers is not in the document]
Comment (anuveyatsu) [reply scripts/edgar.py:6]: ok, let's create a new requirements.txt file with all additional dependencies @gradedSystem

File: scripts/config.py
Comment (gradedSystem) [reply scripts/config.py:1]: I have tested it locally, there is a second pull request here which is also big, there I have fixed every single script and run the Makefile locally and it worked just fine, also added github/workflow

File: scripts/edgar.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (gradedSystem) [reply scripts/edgar.py:6]: @anuveyatsu Sure, already did that on the second pull request which is quite heavy as well, I know that you said to keep it granular, but i had to change lots of things and it didn't make sense to make it granular

File: scripts/iso3166.py (hunk 1 above)
Comment (anuveyatsu) [reply scripts/iso3166.py:2]: Reflected in https://github.com/datasets/country-codes/pull/83/commits/698cc4f4bfe973941f0ec85318b39dd7fcc0d822 cc @gradedSystem

**Pull request merged**